lib/python/treadmill/admin/_ldap.py

```diff
@@ -1068,12 +1068,22 @@ class LdapObject:
     # Allow such names as 'dn', 'ou'
     # pylint: disable=invalid-name
 
+    _operational_attrs = ['createTimestamp', 'modifyTimestamp']
+
     def __init__(self, admin):
         self.admin = admin
 
     def from_entry(self, entry, _dn=None):
         """Converts ldap entry to dict."""
-        return _entry_2_dict(entry, self.schema())
+        obj = _entry_2_dict(entry, self.schema())
+
+        # Add operational attrs if they were requested, those can only be read.
+        # create/modifyTimestamp are single-valued, UTC aware datetime objects.
+        if entry.get('createTimestamp'):
+            obj['create_timestamp'] = entry['createTimestamp'].timestamp()
```

Comment:
I would have prefixed these attributes with `_` to make it clear they are metadata

Reply:
Makes sense, updated.